### PR TITLE
Adds stuff to Biogenerator. STUFF!.

### PR DESCRIPTION
**Pull request by Firecage**
This PR adds some stuff to the biogenerator, which are the following.
1. Janitorial Belt
2. Medical Belt
3. Security Belt
4. Bandolier Belt
5. Bottle of Weed Killer
6. Bottle of Pest Killer
7. Milk Cartons
8. Cream cartons

This also allows you to make x5 of the liquid milk and cream.

As a side note, this also adds bottles of pest killer and weed killer, though obviously this is for the biogenerator.

PS: @Miauw62 cant yell at me now for not adding these stuff.

### CODE REVIEW

**Comment by phil235** [comment]
:+1:

**Comment by Iamgoofball** [comment]
:+1:

**Comment by Delimusca** [comment]
bandolier belt oh fuck yes :+1:

**Comment by WJohn** [comment]
Oh thank god milk cartons. NOBODY ever inserts an actual container into these things so this is a huge plus. :+1:

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -100,6 +100,9 @@
 	..()
 	new /obj/item/device/flashlight/seclite(src)
 
+/obj/item/weapon/storage/belt/security/empty/New()
+	..()
+
```
Comment (Miauw62) [inline code/game/objects/items/weapons/storage/belt.dm]: this doesnt have to be here.

Comment (Firecage) [inline code/game/objects/items/weapons/storage/belt.dm]: Miauw, it does. The default secbelts New() spawns it with a seclite. if I don't put it here, every secbelt you make with the biogenerator would spawn with a seclite.

Comment (RemieRichards) [inline code/game/objects/items/weapons/storage/belt.dm]: This will still spawn it with the seclite.

**Comment by AlexCrimson** [comment]
Universal Enzyme pls

**Comment by Jordie0608** [comment]
Changelog

**Comment by Firecage** [comment]
Good point @Jordie0608 . I shall do that as soon as I return from the bar later.

**Comment by Firecage** [comment]
@Jordie0608 Now that I think about it, isn't it usually wise to make a changelog in a seperate PR so you don't sit with more frequent merge conflicts?

**Comment by RemieRichards** [comment]
@Firecage no, we've had a different system for change logs for a little while now. no conflicts

File: code/modules/hydroponics/biogenerator.dm [hunk 2]
```diff
@@ -135,15 +135,19 @@
 			dat += "<A href='?src=\ref[src];activate=1'>Activate</A><A href='?src=\ref[src];detach=1'>Detach Container</A>"
 			dat += "<h3>Food:</h3>"
 			dat += "<div class='statusDisplay'>"
-			dat += "10 milk: <A href='?src=\ref[src];create=milk;amount=1'>Make</A> ([20/efficiency])<BR>"
-			dat += "10 cream: <A href='?src=\ref[src];create=cream;amount=1'>Make</A> ([30/efficiency])<BR>"
-			dat += "Monkey cube: <A href='?src=\ref[src];create=meat;amount=1'>Make</A> ([250/efficiency])"
+			dat += "10 milk: <A href='?src=\ref[src];create=milk;amount=1'>Make</A><A href='?src=\ref[src];create=milk;amount=5'>x5</A> ([20/efficiency])<BR>"
+			dat += "10 cream: <A href='?src=\ref[src];create=cream;amount=1'>Make</A><A href='?src=\ref[src];create=cream;amount=5'>x5</A> ([30/efficiency])<BR>"
+			dat += "Milk Carton: <A href='?src=\ref[src];create=cmilk;amount=1'>Make</A><A href='?src=\ref[src];create=cmilk;amount=5'>x5</A> ([100/efficiency])<BR>"
+			dat += "Cream Carton: <A href='?src=\ref[src];create=ccream;amount=1'>Make</A><A href='?src=\ref[src];create=ccream;amount=5'>x5</A> ([150/efficiency])<BR>"
```
Comment (tkdrg) [inline code/modules/hydroponics/biogenerator.dm]: This means a carton with 100 cream is 150, half the price needed to get it separately. I suggest doubling it to 300.

File: code/modules/hydroponics/biogenerator.dm [hunk 3]
```diff
@@ -135,15 +135,19 @@
...
-			dat += "10 cream: <A href='?src=\ref[src];create=cream;amount=1'>Make</A> ([30/efficiency])<BR>"
-			dat += "Monkey cube: <A href='?src=\ref[src];create=meat;amount=1'>Make</A> ([250/efficiency])"
+			dat += "10 milk: <A href='?src=\ref[src];create=milk;amount=1'>Make</A><A href='?src=\ref[src];create=milk;amount=5'>x5</A> ([20/efficiency])<BR>"
+			dat += "10 cream: <A href='?src=\ref[src];create=cream;amount=1'>Make</A><A href='?src=\ref[src];create=cream;amount=5'>x5</A> ([30/efficiency])<BR>"
+			dat += "Milk Carton: <A href='?src=\ref[src];create=cmilk;amount=1'>Make</A><A href='?src=\ref[src];create=cmilk;amount=5'>x5</A> ([100/efficiency])<BR>"
+			dat += "Cream Carton: <A href='?src=\ref[src];create=ccream;amount=1'>Make</A><A href='?src=\ref[src];create=ccream;amount=5'>x5</A> ([150/efficiency])<BR>"
+			dat += "Monkey cube: <A href='?src=\ref[src];create=meat;amount=1'>Make</A><A href='?src=\ref[src];create=meat;amount=5'>x5</A> ([250/efficiency])"
 			dat += "</div>"
-			dat += "<h3>Nutrients:</h3>"
+			dat += "<h3>Botany Chemicles:</h3>"
```
Comment (tkdrg) [inline code/modules/hydroponics/biogenerator.dm]: `Chemicals`, not `chemicles`.

File: code/modules/hydroponics/biogenerator.dm [hunk 4]
```diff
@@ -135,15 +135,19 @@
...
 			dat += "<div class='statusDisplay'>"
 			dat += "E-Z-Nutrient: <A href='?src=\ref[src];create=ez;amount=1'>Make</A><A href='?src=\ref[src];create=ez;amount=5'>x5</A> ([10/efficiency])<BR>"
 			dat += "Left 4 Zed: <A href='?src=\ref[src];create=l4z;amount=1'>Make</A><A href='?src=\ref[src];create=l4z;amount=5'>x5</A> ([20/efficiency])<BR>"
 			dat += "Robust Harvest: <A href='?src=\ref[src];create=rh;amount=1'>Make</A><A href='?src=\ref[src];create=rh;amount=5'>x5</A> ([25/efficiency])<BR>"
+			dat += "Weed Killer: <A href='?src=\ref[src];create=wk;amount=1'>Make</A><A href='?src=\ref[src];create=wk;amount=5'>x5</A> ([25/efficiency])<BR>"
+			dat += "Pest Killer: <A href='?src=\ref[src];create=pk;amount=1'>Make</A><A href='?src=\ref[src];create=pk;amount=5'>x5</A> ([25/efficiency])<BR>"
```
Comment (tkdrg) [inline code/modules/hydroponics/biogenerator.dm]: These 2 are a bit cheap IMO. I'd bump both of them up to 50.

**Comment by Firecage** [comment]
Okay @tkdrg Its done.

**Pull request merged**